### PR TITLE
Request a small amount of ephemeral storage.

### PR DESCRIPTION
**Pull request by cupcicm**
So that kubernetes can store logs about our container.

### CODE REVIEW

File: charts/warpstream-agent/CHANGELOG.md [hunk 1]
```diff
@@ -148,7 +155,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - update warpstream agent version to v558
-- 
+-
```
Comment (epot) [inline charts/warpstream-agent/CHANGELOG.md:158]: remove this `-` ?

File: charts/warpstream-agent/values.yaml [hunk 2]
```diff
@@ -79,8 +79,10 @@ resources:
   requests:
     cpu: 2
     memory: 8Gi
+    ephemeral-storage: "100Mi"
```
Comment (epot) [inline charts/warpstream-agent/values.yaml:82]: let's put a fat comment here to make sure people don't think we are storing data on disk?

**Review by epot — APPROVED** [review]

File: charts/warpstream-agent/values.yaml (hunk 2 above)
Comment (richardartoul) [reply charts/warpstream-agent/values.yaml:82]: +1

File: charts/warpstream-agent/CHANGELOG.md (hunk 1 above)
Comment (cupcicm) [reply charts/warpstream-agent/CHANGELOG.md:158]: Done.

**Pull request merged**